### ISSUE TITLE
Window.Activated Event occurs after the WinUI Desktop Window has already been closed

### ISSUE THREAD

**Issue by tom-huntington** [microsoft/microsoft-ui-xaml#6662]
### Describe the bug

When exiting the app though the title bar close button, the debugger will stop on a exception

>Exception thrown at 0x75EDEC52 (KernelBase.dll) in App1.exe: WinRT originate error - 0x800710DD : 'The WinUI Desktop Window object has already been closed.'.
Exception thrown at 0x75EDEC52 (KernelBase.dll) in App1.exe: WinRT originate error - 0x80000013 : 'The given object has already been closed / disposed and may no longer be used.'.
Exception thrown at 0x75EDEC52 (KernelBase.dll) in App1.exe: WinRT originate error - 0x80000013 : 'The given object has already been closed / disposed and may no longer be used.'.
Exception thrown at 0x75EDEC52 (KernelBase.dll) in App1.exe: WinRT originate error - 0x800710DD : 'The WinUI Desktop Window object has already been closed.'.

This is mildly annoying for development. Is there a way to detect this and return early?

### Steps to reproduce the bug

To reproduce just add a `WindowActivated` event to the Blank App template with the following handler:

```
void winrt::App1::implementation::MainWindow::Window_Activated(winrt::Windows::Foundation::IInspectable const& sender, winrt::Microsoft::UI::Xaml::WindowActivatedEventArgs const& args)
{
    auto content = Content();
}
```

### Windows app type

- [ ] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

Insider Build (xxxxx)

**Comment by tom-huntington** [comment]
I solved this by upgrading to the Windows App SDK

**Comment by tom-huntington** [comment]
The way to solve this is by handling the `Window.Closed` event. You cant unsubscribe you event handler if you cant figure out how.

Although, I just used a boolean member variable to return early from my event handler.

Feel free to close

**Comment by Slion** [comment]
I have the same issue in my app. Though I'm using release 1.0 AFAICT.
In my case removing the activate handler did not help.

**Comment by tpoint75** [comment]
I have the same bug gettting an exception on exiting the app: im sure it came by updating on of these packages, but I don't know the version in witchj it was ok.

It occurs after leaving the void MainWindow::OnClosed() function.
The it runs 

I can debug it until base.h:
 WINRT_IMPL_NOINLINE void unconditional_release_ref() noexcept
 {
     std::exchange(m_ptr, {})->Release();
 }

Microsoft.Graphics.Win2D: 1.1.0-preview1
Microsoft.Windows.CppWinRT: 2.0.230706.1
Microsoft.Windows.ImplementationLibrary: 1.0.230629.1
Microsoft.Windows.SDK.BuildTools: 10.0.25926-preview
Microsoft.WindowsAppSDK: 1.3.230724000

Ausnahme ausgelöst bei 0x00007FFF68B0CF19 (KernelBase.dll) in Harp.exe: WinRT originate error - 0x80000013 : 'The given object has already been closed / disposed and may no longer be used.'.
Ausnahme ausgelöst bei 0x00007FFF68B0CF19 (KernelBase.dll) in Harp.exe: 0x40080202: WinRT transform error (Parameter: 0x000000008000000B, 0x0000000080070490, 0x0000000000000019, 0x000000A26FCFD810).
Ausnahme ausgelöst bei 0x00007FFF68B0CF19 (KernelBase.dll) in Harp.exe: 0x40080202: WinRT transform error (Parameter: 0x000000008000000B, 0x0000000080070490, 0x0000000000000019, 0x000000A26FCFD810).
Microsoft.UI.Xaml.dll!00007FFE804420B7: CallContext:[\ImageDecodeActivity] 8001010E - RPC_E_WRONG_THREAD
Ausnahme ausgelöst bei 0x00007FFF68B0CF19 (KernelBase.dll) in Harp.exe: WinRT originate error - 0x80000013 : 'The given object has already been closed / disposed and may no longer be used.'.
onecoreuap\windows\moderncore\inputv2\inputhost\components\dragdrop\lib\closeableobject.cpp(31)\Microsoft.UI.Input.dll!00007FFE69A6F1A1: (caller: 00007FFE69A6F022) ReturnHr(1) tid(4588) 80000013 Das Objekt wurde geschlossen.
Ausnahme ausgelöst bei 0x00007FFE69F37F22 (Microsoft.ui.xaml.dll) in Harp.exe: 0xC0000005: Zugriffsverletzung beim Lesen an Position 0x00000000000001A8.

**Comment by Ajun11** [comment]
Hello, my friends, my program has had a similar problem as yours. During my check, I found that this problem is from a basic idea: free your variable, just like some friends have chated in this issue, someone use "new" , but maybe they are not free this variable. Just like mine, I have defined a variable:` static inline winrt::Microsoft::UI::Xaml::Controls::NavigationView root_page_navView{ nullptr };`in `MainPage.xaml.h`,  I need to solve my program's work,  . First time, I didn't free this variable in `App.xaml.h` function `    void App::OnWindowClosed(winrt::Windows::Foundation::IInspectable const& sender, Microsoft::UI::Xaml::WindowEventArgs const& args)
`, what we need to do is to free this variable in this function, just like using nullptr or others.
